profiler.py

```diff
@@ -0,0 +1,96 @@
+import json
+
+import numpy as np
+import pandas as pd
+
+# create a test epw with fixed values to ascertain the impact of different things in the ExternalComfort modules
+from ladybugtools_toolkit.ladybug_extension.epw import (
+    EPW,
+    Location,
+    Path,
+    epw_from_dataframe,
+    epw_to_dataframe,
+)
+from ladybugtools_toolkit.new_external_comfort.material import get_material
+from ladybugtools_toolkit.new_external_comfort.shelter import (
+    SENSOR_LOCATION,
+    Point3D,
+    Shelter,
+    Shelters,
+    TreeSpecies,
+    get_tree_shelter,
+)
+from ladybugtools_toolkit.new_external_comfort.simulate import (
+    SimulationResult,
+    collection_from_series,
+    collection_to_series,
+)
+from ladybugtools_toolkit.new_external_comfort.typology import Typologies
+
+print("Generating EPW file ...")
+epw_file = r"C:\Users\tgerrish\BuroHappold\Sustainability and Physics - epws\ITA_SD_Cape.Bellavista.165500_TMYx.2004-2018.epw"
```

Comment:
If this file is necessary, Should replace file paths with relative ones, and remove references to BuroHappold.